pkgs/development/compilers/llvm/13/libcxx/default.nix

```diff
@@ -41,7 +41,13 @@ stdenv.mkDerivation rec {
       "-DLIBCXX_ENABLE_THREADS=OFF"
       "-DLIBCXX_ENABLE_FILESYSTEM=OFF"
       "-DLIBCXX_ENABLE_EXCEPTIONS=OFF"
-    ] ++ lib.optional (!enableShared) "-DLIBCXX_ENABLE_SHARED=OFF";
+    ]
+    ++ lib.optional (!enableShared) "-DLIBCXX_ENABLE_SHARED=OFF"
+    ++ lib.optionals (!headersOnly && libcxxabi.semi-static) [
+      "-DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=TRUE"
+      "-DLIBCXX_CXX_ABI_LIBRARY_PATH=${libcxxabi}/lib"
+    ] ++ lib.optional (!headersOnly && libcxxabi.useLLVMUnwinder)
+      "-DLIBCXXABI_USE_LLVM_UNWINDER=ON";
```

Comment:
Curious about this; `libcxx/CMakeLists.txt` claims this is only about linking the test cases which we don't run.

Reply:
I got link errors without this, I wouldn't be surprised if it's only needed when `LIBCXX_ENABLE_STATIC_ABI_LIBRARY` (which is narrowly tested).

Reply:
In which version, @sternenseemann I don't see that in LLVM 13's `libcxx/CMakeLists.txt`.

Reply:
I was looking at LLVM 9, but the comment is still there on `main`: https://github.com/llvm/llvm-project/blob/7e3ef7dcd2b8201dce671237857f04a1d527d580/libcxx/CMakeLists.txt#L297-L298